### DISCUSSION TITLE
Level Loading Operations

### DISCUSSION THREAD

**Discussion by RKBusinessAnalyst** [frePPLe/frepple#535] [Q&A] [answered]
This is probably a fairly simple question but I am attempting to level load operations so that they are done evenly across the available time period instead of using large amount of capacity up front. Something similar to the image below. I am aware that constraining the capacity will "level out" the manufacturing but I want to be able to assign it with percentages or manually in a worst case scenario.
![image](https://user-images.githubusercontent.com/108491411/176765056-868367ae-7229-4bb9-b943-ffe85721b968.png)

**Comment by jdetaeye** [comment, marked as answer]
There is no standard feature to achieve this.

With a customization, you can solve it with 2 planning runs: in first planning run you run an capacity unconstrained plan to establish the average load / blue line and generate a capacity calendar for the resources. In a second planning you then run a capacity constrained plan using the average load as the capacity limit.
